ci-operator/step-registry/rhcs/e2e/setup/rhcs-e2e-setup-ref.yaml

```diff
@@ -0,0 +1,33 @@
+ref:
+  as: rhcs-e2e-setup
+  from_image:
+    namespace: ci
+    name: rhcs-tf-e2e
+    tag: latest
+  grace_period: 10m
+  commands: rhcs-e2e-setup-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 300Mi
+  timeout: 2h0m0s
+  env:
```

Comment:
support for different provider (local or terraform-site) ?

Reply:
No, it's a future enhancement.